build/yamls/flow-aggregator/flow-aggregator.yml

```diff
@@ -0,0 +1,80 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: flow-aggregator
+  name: flow-aggregator
+  namespace: flow-aggregator
+spec:
+  selector:
+    app: flow-aggregator
+  type: NodePort
+  ports:
+  - port: 4739
+    protocol: UDP
+    targetPort: 4739
+---
+apiVersion: v1
+data:
+  flow-aggregator.conf: |
+    # Provide the flow collector address as string with format <IP>:<port>[:<proto>], where proto is tcp or udp.
+    # If no L4 transport proto is given, we consider tcp as default.
+    # Defaults to "".
+    flowCollectorAddr: "10.96.28.149:4739:udp"
+
+    # Provide flow export frequency, which is the number of seconds elapsed before flow aggregator exports flow
+    # records to the flow collector.
+    # Flow export frequency should be greater than or equal to 1.
+    # Defaults to "60s".
+    flowExportFrequency: 60s
+kind: ConfigMap
+metadata:
+  annotations: {}
+  labels:
+    app: flow-aggregator
+  name: flow-aggregator-configmap
+  namespace: flow-aggregator
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: flow-aggregator
+  labels:
+    app: flow-aggregator
+  namespace: flow-aggregator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: flow-aggregator
+  template:
+    metadata:
+      labels:
+        app: flow-aggregator
+    spec:
+      containers:
+      - args:
+        - --config
+        - /etc/flow-aggregator/flow-aggregator.conf
+        - --logtostderr=false
+        - --log_dir=/var/log/flowaggregator
+        - --alsologtostderr
+        - --log_file_max_size=100
+        - --log_file_max_num=4
+        - --v=0
+        command:
+          - flow-aggregator
+        name: flow-aggregator
+        image: dreamtalen/flow-aggregator
```

Comment:
Are you planning to use Antrea org in dockerhub registry to maintain the image?

Reply:
Yes, changed the image name to antrea/flow-aggregator